frontend/src/components/layout/PageWrapper/PageWrapper.tsx

```diff
@@ -58,3 +58,28 @@ export const PageWrapper: React.FC<PageWrapperProps> = ({
 
   return <BaseContentWrapper>{children}</BaseContentWrapper>;
 };
+
+const BaseContentContainer = styled.main`
+  width: 100%;
+  margin: 0 auto;
+  background-color: #fff;
+  padding: 0 ${pxToRem(24)};
+  max-width: ${pxToRem(1800)};
+`;
+
+const ContentWrapper = styled.div`
+  width: 100%;
+  height: 100%;
+  padding: 0 ${pxToRem(24)};
+
+  @media (min-width: 768px) {
+    padding: 0 ${pxToRem(32)};
+  }
+  @media (min-width: 1280px) {
+    padding: 0 ${pxToRem(48)};
```

Comment:
These breakpoint pixel values can be replaced with the breakpoints obj in styled-theme.ts